pypeit/core/combine.py

```diff
@@ -84,7 +84,7 @@ def weighted_combine(weights, sci_list, var_list, inmask_stack,
               images with shape (nspec, nspat)
             - var_list_out: list: The list of ndarray propagated
               variance images with shape (nspec, nspat)
-            - outmask: bool ndarray, shape (nspec, nspat): Mask for
+            - gpm: bool ndarray, shape (nspec, nspat): Mask for
```

Comment:
Can we write "good pixel mask in the docs to make it more clear"

Reply:
edited